bids/modeling/statsmodels.py

```diff
@@ -263,11 +264,11 @@ class BIDSStatsModelsNode:
         overridden if one is passed when run() is called on a node.
     """
 
-    def __init__(self, level, name, transformations=None, model=None,
-                 contrasts=None, dummy_contrasts=False, group_by=None):
+    def __init__(self, level, name, model, group_by, transformations=None,
```

Comment:
I made `model` and `group_by` mandatory, because they are mandatory in the model spec